<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:rgb(3, 148, 15)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dealerCard1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerHeaderLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dealer</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerHeaderLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="spushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(216, 216, 216)
}</string>
    </property>
    <property name="text">
     <string>Shuffle Start/Over</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitMeButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>740</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(216, 216, 216)
}</string>
    </property>
    <property name="text">
     <string>Hit Me!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="standButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>740</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:rgb(216, 216, 216)
}</string>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMessageBox{
	background:rgb(214, 214, 214)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>45</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>400</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
